Tests/FunctionalTests/helpermethod.ts

```diff
@@ -40,4 +40,39 @@ export async function SearchAttribute(page:Page, attributeName: string)
     await page.keyboard.press('ArrowDown');
     await page.keyboard.press('Enter'); 
     page.waitForLoadState('domcontentloaded');
+  }
+
+  export async function SearchAttributeSecondRow(page:Page, attributeName: string)
+  {
+    await page.fill(pageObjectsConfig.inputSearchFieldSelectorSecondRow,"");   
+    await page.fill(pageObjectsConfig.inputSearchFieldSelectorSecondRow,attributeName);
+    await page.keyboard.press('Backspace');    
+    await page.keyboard.press('ArrowDown');
+    await page.keyboard.press('Enter'); 
+    page.waitForLoadState('domcontentloaded');
+  }
+
+//<summary>
+// Get the file Size in bytes.
+//</summary>
+//<param> filesize </param>
+
+  export function  GetFileSizeInBytes(fileSize: string)
+  {
+        let fileSizeChar=fileSize.split(' ');       
+
+        if (fileSizeChar[1]=='KB')
```

Comment:
Would a switch statement be better here?

Reply:
Done Changes. Thanks.